frontend/src/histogram/ui/ChartPage.vue

```diff
@@ -5,7 +5,7 @@
             <div class="visual-dl-chart-page-box">
                 <ui-chart
                     v-for="tagInfo in filteredPageList"
-                    :key="tagInfo.run + tagInfo.tag.displayName"
+                    :key="tagInfo.tag"
```

Comment:
We should avoid using non-primitive value as the key. Vue will complain this. 
Let's just use Varun's index suggestion.